app/views/home/companies.md

```diff
@@ -0,0 +1,56 @@
+> Audience: Employees of a company that use and depend on open source software (a.k.a. every company that employs developers). Specifically, this page should be a link that people can send their bosses/coworkers/employees to help them learn what open source is, why it's so important to their company, and why their employees should take time to participate in Open Source Friday.
+
+> What are you asking me to do?
+
+Take one day this week to invest in the open source software you use and the community of people that make it great.
+
+> Why should I care?
+
+## Your business depends on it
+
+> What are the reasons companies should care about their employees contributing to and maintaining open source?
+
+- **Improve your technology** - Open source infrastructure is essential to running businesses effectively. Take control rather than relying entirely on others to maintain the software your business depends on.
+
+- **Reduce overhead** - Small, regular contributions to open source allow you to stop maintaining internal versions of open source software in favor of the projects maintaining your patches
+
+- **Improve Recruiting** - Build the reputation of your organization aiding community outreach efforts and hiring.
```

Comment:
```Improve recruiting``` should be something more emotional, even like ```Build your company's reputation```. I don't think we need to explicitly say hiring/recruiting but rather focus on talent/reputation.

There are two benefits here, too: 1) letting your engineers create public artifacts improves *their* reputation, making you the cool company with fancy famous engineers, and 2) when your company software is built in public, you'll attract relevant developer talent.